docs/Displaying a User Interface/Adaptive Interface Capabilities/index.md

```diff
@@ -160,6 +179,11 @@ let supportsSubscriptions = sdlManager.systemCapabilityManager.supportsSubscript
 boolean supportsSubscriptions = sdlManager.getSystemCapabilityManager().supportsSubscriptions();
 ```
 !@
+@![javascript]
+```js
+const supportsSubscriptions = sdlManager.getSystemCapabilityManager().supportsSubscriptions();
```

Comment:
JS Suite doesn't have this method. Will just have to remove this and say it isn't supported. Must have been a recent addition for the other libraries